RandomNumber.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+public class GetValue
+{
+    public string name { get; } = "Bat";
+}
+class RandomNumber
+{
+    private string ran;
+    public RandomNumber()
```

Comment:
use a parameterized constructor to take in a string value and assign it to the private field within the class